pkg/drivers/vfkit/vfkit.go

```diff
@@ -256,9 +251,10 @@ func (d *Driver) startVfkit(socketPath string) error {
 		"--memory", fmt.Sprintf("%d", d.Memory),
 		"--cpus", fmt.Sprintf("%d", d.CPU),
 		"--restful-uri", fmt.Sprintf("unix://%s", d.sockfilePath()))
-	var isoPath = filepath.Join(machineDir, isoFilename)
+
+	efiPath := d.ResolveStorePath(efiFileName)
```

Comment:
The name "efiPath" is slightly disambigous, here it is referring to the variable store (and not e.g. firmware)

Reply:
Changed to efiVarsPath, matching new constant.